tests/util.h

```diff
@@ -29,7 +29,7 @@ double calculate_snr (float *data, int len, int expected_peaks) ;
 
 const char * get_cpu_name (void) ;
 
-#if OS_IS_WIN32
+#ifdef WIN32
```

Comment:
Using `#ifdef` is less safe than using `#if`. If the mechanism that defines a variable used like this goes wrong, it can fail silently.

The variable `OS_IS_WIN32` is defined to be either `0` or `1`. If the mechanism to define it goes wrong, you get a compile error.

Reply:
I agree here and like that error too (used in our projects too). However in this case the complexity in configuring this could be easily replaced by existing macros which are defined for exactly this purpose. I could/should enhance this to `if defined(_WIN32) || defined(__WIN32__) || defined(WIN32)` to catch more compilers (I think `_WIN32` is the most correct one, the rest is compatibility)

As this is in tests though I don't really care which way it is done. Your call

Reply:
I updated this to use the old define. As this is test code it doesn't matter if configure is required.